ydb/core/tx/schemeshard/schemeshard__operation_move_index.cpp

```diff
@@ -596,14 +596,17 @@ TVector<ISubOperation::TPtr> CreateConsistentMoveIndex(TOperationId nextId, cons
 
     result.push_back(CreateMoveTableIndex(NextPartId(nextId, result), MoveTableIndexTask(srcIndexPath, dstIndexPath)));
 
-    TString srcImplTableName = srcIndexPath.Base()->GetChildren().begin()->first;
-    TPath srcImplTable = srcIndexPath.Child(srcImplTableName);
+    for(const auto& implTable : srcIndexPath.Base()->GetChildren()) {
```

Comment:
```suggestion
for (const auto& implTable : srcIndexPath.Base()->GetChildren()) {
```

А можно и сразу распаковать, чтобы не смотреть в `first` и `second`.

Reply:
https://github.com/ydb-platform/ydb/pull/18011